bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,223 @@
+| fineDustCnt00_3           | Number:Dimensionless |    | Fine Dust >0,3 µm                                                   |
+| fineDustCnt00_5           | Number:Dimensionless |    | Fine Dust >0,5 µm                                                   |
+| fineDustCnt01             | Number:Dimensionless |    | Fine Dust >1 µm                                                     |
+| fineDustCnt02_5           | Number:Dimensionless |    | Fine Dust >2,5 µm                                                   |
+| fineDustCnt05             | Number:Dimensionless |    | Fine Dust >5 µm                                                     |
+| fineDustCnt10             | Number:Dimensionless |    | Fine Dust >10 µm                                                    |
+| co                        | Number:Dimensionless |    | CO concentration                                                    |
+| co2                       | Number               |    | CO₂ concentration                                                   |
+| dCO2dt                    | Number               |    | Change of CO₂ concentration                                         |
+| dHdt                      | Number               |    | Change of Humidity                                                  |
+| dewpt                     | Number:Temperature   |    | Dew Point                                                           |
+| doorEvent                 | Number               |    | Door Event (experimental, might not work reliably)                  |
+| health                    | Number:Dimensionless |    | Health Index (0 to 1000, -200 for gas alarm, -800 for fire alarm)   |
+| humidityRelative          | Number:Dimensionless |    | Humidity in percent                                                 |
+| humidityAbsolute          | Number               |    | Absolute Humidity                                                   |
+| measureTime               | Number               |    | Milliseconds needed for measurement                                 |
+| no2                       | Number               |    | NO₂ concentration                                                   |
+| o3                        | Number               |    | Ozone (O₃) concentration                                            |
+| o2                        | Number:Dimensionless |    | Oxygen (O₂) concentration                                           |
+| performance               | Number:Dimensionless |    | Performance Index (0 to 1000)                                       |
+| fineDustConc01            | Number               |    | Fine Dust concentration >1 µm                                       |
+| fineDustConc02_5          | Number               |    | Fine Dust concentration >2.5 µm                                     |
+| fineDustConc10            | Number               |    | Fine Dust concentration >10 µm             fni                      |
```

Comment:
```suggestion
| fineDustConc10            | Number               |    | Fine Dust concentration >10 µm                                      |
```

Reply:
Is this a typo?

Reply:
I don't know. What is the purpose of the "fni"?

Reply:
Sorry, I had understood that you inserted it. But actually it was _my_ typo, and I removed it now.